test/sql/function/string/test_format_extensions.test

```diff
@@ -16,6 +16,11 @@ select format('{:d}', 123456789)
 ----
 123456789
 
+query I
+select printf('%,d', 123456789123456789123456789::HUGEINT)
+----
+123,456,789,123,456,789,123,456,789
```

Comment:
Can we add a test for a negative hugeint, as well as for the min/max values of hugeint? (these can be obtained from `select hugeint, uhugeint from test_all_types();`)

Reply:
Thanks for advice! The negative case is wrong and I've added a fix.